samples/dashboards/demo/vertical/demo.html

```diff
@@ -0,0 +1,117 @@
+            planet’s
+            delicate balance, even slight temperature changes can cause big disruptions.
+        </p>
+        <p>
+            <b>- 1980-2019:</b> From the 1980s onward, the pace of temperature increase accelerated dramatically.
+            By the
+            2010s, the global temperature had increased by 0.8°C. This steady increase is directly linked to human
+            activity,
+            particularly the burning of fossil fuels like coal, oil, and natural gas.
+        </p>
+        <div class="cell" id="temp-chart"></div>
+    </span>
+
+    <span>
+        <h2>2. CO2 Concentration (ppm)</h2>
+        CO2, or carbon dioxide, is one of the main greenhouse gases responsible for trapping heat in the Earth's
+        atmosphere. This column shows the concentration of CO2 in parts per million (ppm).
+        <p>
+            <b>- 1920-1950:</b> CO2 levels hovered around 305-315 ppm during the early 20th century. While industrial
+            activity was
+            increasing, CO2 emissions were not yet significant enough to cause noticeable shifts in climate patterns.
+        </p>
+        <p>
+            <b>- 1950-1980:</b> After the 1950s, CO2 levels started to climb, reaching 327 ppm by the 1970s. The
+            post-war boom in
+            industrialization and the widespread use of cars played a large role in this increase.
+        </p>
+        <p>
+            <b>- 1980-2019:</b> CO2 levels rose dramatically, reaching 400 ppm by the 2010s. The combination of
+            fossil fuel use,
+            deforestation, and industrial emissions led to this unprecedented rise, which is a major contributor to
+            the warming of the planet.
+        </p>
+        <div id="co2-chart"></div>
+    </span>
+
+    <span>
+        <h2>3. Sea Level Rise (cm)</h2>
+        Sea levels rise for two main reasons: melting ice from glaciers and ice sheets, and the expansion of seawater as
+        it warms (thermal expansion). This column shows how much sea levels have risen globally.
+        <p>
+            <b>- 1920-1950:</b> For the first half of the century, sea levels were relatively stable, with only a slight
+            rise of 0.2 cm by the 1930s. The changes were minimal because the Earth’s temperature hadn’t risen
+            significantly yet.
+        </p>
+        <p>
+            <b>- 1950-1980:</b> As temperatures started to rise, glaciers began to melt and oceans expanded. By the
+            1970s, the sea level had risen by about 2 cm. Though small, this rise had started to impact coastal regions.
+        </p>
+        <p>
+            <b>- 1980-2019:</b> From the 1980s onwards, the pace of sea level rise accelerated dramatically. By the
+            2010s, sea levels had risen by 8 cm compared to early 20th-century levels, leading to coastal flooding and
+            erosion in many parts of the world.
+        </p>
+        <div id="sea-chart"></div>
```

Comment:
Position to the left of the text. Should take up 50% of available width, with some margin to the left.

Add 20px padding to the div, and use same bg color as chart. Add border-radius: 5px.

Reply:
Can't see that the additional styling is applied here? Padding, background-color and border-radius?